src/gui/systray.cpp

```diff
@@ -121,8 +121,9 @@ void Systray::showMessage(const QString &title, const QString &message, MessageI
 {
 #ifdef USE_FDO_NOTIFICATIONS
     if (QDBusInterface(NOTIFICATIONS_SERVICE, NOTIFICATIONS_PATH, NOTIFICATIONS_IFACE).isValid()) {
+        const QVariantMap hints = {{QStringLiteral("desktop-entry"), QStringLiteral("com.nextcloud.desktopclient.nextcloud")}};
```

Comment:
I think you should use https://github.com/nextcloud/desktop/blob/78776f81e4e281010c3223715d9ce27ba7b4590f/NEXTCLOUD.cmake#L13 here instead of hardcoding it?